app/lib/features/events/pages/event_page.dart

```diff
@@ -15,28 +14,60 @@ import 'package:flutter_easyloading/flutter_easyloading.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:go_router/go_router.dart';
 import 'package:jiffy/jiffy.dart';
+import 'package:material_segmented_control/material_segmented_control.dart';
 
-class CalendarEventPage extends ConsumerWidget {
+class CalendarEventPage extends ConsumerStatefulWidget {
   final String calendarId;
 
   const CalendarEventPage({super.key, required this.calendarId});
 
+  @override
+  ConsumerState<CalendarEventPage> createState() => _CalendarEventPageState();
+}
+
+class _CalendarEventPageState extends ConsumerState<CalendarEventPage> {
+  RsvpStatusTag? rsvpStatus;
+
+  @override
+  void initState() {
+    super.initState();
+    AsyncValue<OptionRsvpStatus> myRsvpStatus =
+        ref.read(myRsvpStatusProvider(widget.calendarId));
```

Comment:
why replace the watch with a singular read? This doesn't sound like a good idea..

Reply:
As you see in below code, `setState()` is called when `Yes`, `Maybe` or `No` is tapped.
As we know, `setState()` is meaningless about `ref.watch()`.